system-tests/project-fixtures/next/cypress/support/component-index.html

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Above file packages/scaffold-config/test/unit/frameworks.spec.ts was obsolete (code was moved to `data-context` and tests added there) so I deleted it. The reason we did not notice was because `scaffold-config` was not running on CI (now it is as of this PR).